client/src/App.js

```diff
@@ -2,34 +2,52 @@ import React, { useEffect, useState } from "react";
 import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
 import { MuiThemeProvider } from "@material-ui/core";
 import { theme } from "./themes/theme";
-import CarRentalPage from "./pages/CarRentalPage";
+import { makeStyles } from "@material-ui/core/styles";
 import Navbar from "./components/Navbar";
 import UserPage from "./components/Userpage";
 import SearchPage from "./pages/SearchPage";
 import SignUp from "./components/SignUp";
 import SignIn from "./components/SignIn";
+import CarRental from "./pages/CarRentalPage";
 import ProtectedRoute from "./components/ProtectedRoute";
 import Explore from "./components/Explore";
 import { ShoppingCartProvider } from "./components/ShoppingCartContext";
 
+const appStyles = makeStyles((theme) => ({
+	container: {
+		position: "fixed",
+		width: "100%",
+		height: "100%",
+		top: "0",
+		right: "0",
+		bottom: "0",
+		margin: "auto",
+		backgroundColor: "rgba(0,0,0, 0.5)",
+		display: "flex",
+		justifyContent: "center",
+		alignItems: "center",
+	},
+}));
+
 function App() {
-  const [loggedIn, setLoggedIn] = useState(false);
+	const [loggedIn, setLoggedIn] = useState(false);
+	const classes = appStyles(theme);
 
-  useEffect(() => {
-    checkLoggedIn();
-  });
+	useEffect(() => {
+		checkLoggedIn();
+	});
 
-  const checkLoggedIn = () => {
-    if (localStorage.loggedIn) {
-      return setLoggedIn(true);
-    } else {
-      return setLoggedIn(false);
-    }
-  };
+	const checkLoggedIn = () => {
+		if (localStorage.loggedIn) {
+			return setLoggedIn(true);
+		} else {
+			return setLoggedIn(false);
+		}
+	};
 
-  const handleModalExit = () => {
-    setLoggedIn(true);
-  };
+	const handleModalExit = () => {
+		setLoggedIn(true);
+	};
```

Comment:
okay as a team we'll need to clarify our prettier settings. I think default is 2 space tabs but yours is for some reason 4 space tabs

Reply:
Yeah i changed my spacing back to 2. Thanks!